AnalyticalModels/README.md

```diff
@@ -1,32 +1,37 @@
 In this repo we have the tools for constructing shapes in the kt X kl plane analitically (this framework is extensible to c2, cg, c2g and more when necessary).
+The input elements are described inside the folder "data"
 
 ===========================================================================================
 
-In the file "python nonResonant_test.py" there are functions to apply analytical formulas for signal efficiencies in bins. 
+In the file "test/nonResonant_test_v(0/JHEP).py" we have a template how to use the above file to calculate event-by-event weights. 
 
-The bins are defined in the gen-level variables as:
+      cd HHStatAnalysis/AnalyticalModels/test
+      python nonResonant_test_v(0/JHEP).py --kl 1 --kt 1
```

Comment:
I personally prefer ready-to-be-copy-pasted command lines, could you chose one case or the other ?